From: Andre Mauricio Zelak <andre.zelak@windriver.com>
Date: Mon, 12 Jun 2023 17:51:10 -0300
Subject: [PATCH 32/61] phc2sys: Fix regression in the automatic mode.

Commit ac7d69bbc476 ("pmc_agent: Convert the method that queries the
port properties.") had the well meant intention of the cleaning up the
error code semantics of the port properties query function.  However,
that commit mixed up the normal, external semantics of zero meaning
success with the internal semantics where zero is an error.  Correct
the issue by replacing the hard coded number with the proper macro.

Signed-off-by: Richard Cochran <richardcochran@gmail.com>
Fixes: ac7d69bbc476 ("pmc_agent: Convert the method that queries the port properties.")

[commit 0fb1be2f5c4d6905f33a2b1c31e7496d52296748 upstream]
Signed-off-by: Andre Mauricio Zelak <andre.zelak@windriver.com>
---
 pmc_agent.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pmc_agent.c b/pmc_agent.c
index 6e6627d..623f300 100644
--- a/pmc_agent.c
+++ b/pmc_agent.c
@@ -338,7 +338,7 @@ int pmc_agent_query_port_properties(struct pmc_agent *node, int timeout,
 		iface[len] = '\0';
 
 		msg_put(msg);
-		res = 0;
+		res = RUN_PMC_OKAY;
 		break;
 	}
 out:
